MediaRecorder.bs

```diff
@@ -410,6 +411,46 @@ interface MediaRecorder : EventTarget {
   </table>
   </dd>
 
+  <dt><dfn method for="MediaRecorder"><code>replaceStream(MediaStream newStream)</code></dfn></dt>
+  <dd>
+  When a {{MediaRecorder}} object&#8217;s {{replaceStream()}} method is invoked,
+  the UA MUST run the following steps:
+  <ol>
+    <li>Let |recorder| be the {{MediaRecorder}} object on which the method was
+    invoked.</li>
+    <li>Let |newStream| be the method's first argument.</li>
+    <li>Let |newTracks| be the set of {{live}} tracks in |newStream|'s
+    [=track set=].</li>
+    <li>Let |stream| be the value of |recorder|'s {{stream}} attribute.</li>
+    <li>Let |tracks| be the set of {{live}} tracks in |stream|'s
+    [=track set=].</li>
+    <li>If the number of audio tracks in |newTracks| is different than the
+    number of audio tracks in |tracks|, or the number of video tracks in
+    |newTracks| is different than the number of video tracks in |tracks|, return
+    a promise {{rejected}} with a newly created {{InvalidModificationError}} and
+    abort these steps.</li>
+    <li>Let |p| be a new promise and run the following steps in parallel:</li>
+    <ol>
+      <li>If data is currently being gathered, replace the source stream being
```

Comment:
@jan-ivar If media is not being recorded this step is skipped, and we immediately jump back and replace the stream attribute and resolve the promise.

Reply:
What should occur when the use case is one audio track with multiple video tracks being replaced? Should a new `MediaStream` object be created for each video track replacement with the single audio track being added to the replacement `MediaStream` from the replaced `MediaStream`? 

What are the reasons for proposing `replaceStream()` instead of `replaceTrack()`?